### PR TITLE
InfluxDB Plugin

### PR DESCRIPTION
**Pull request by hotzen**
started as a fork of influxdata which is lacking too much and I had to give it a proper name (influxdata is the vendor/company that also procudes the product/database infuxdb)

### CODE REVIEW

File: influxdb/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  database = "smarthome"
+  # retention-policy = ""
+</pre>
+
+### plugin.conf
+you can setup global tags and fields (JSON encoded)
+<pre>
+[influxdb]
+    class_name = InfluxData
+    class_path = plugins.influxdata
```
Comment (ohinckel) [inline influxdb/README.md]: The `plugins.influxdata` should be `plugins.influxdb`.

Comment (ohinckel) [inline influxdb/README.md]: Additionally the `InfluxData` should be `InfluxDb`.

File: influxdb/__init__.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
+
+
+#!/usr/bin/env python3
```
Comment (ohinckel) [inline influxdb/__init__.py]: I don't know if these emptly line is a problem, but could be one with the shebang.

File: influxdb/__init__.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        self.item_config = {}
+
+    def run(self):
+        self.alive = True
+
+    def stop(self):
+        self.alive = False
+
+    def parse_item(self, item):
+        if self.keyword in item.conf or 'influxdb_name' in item.conf or 'influxdb_tags' in item.conf or 'influxdb_fields' in item.conf:
```
Comment (ohinckel) [inline influxdb/__init__.py]: Checking and getting item config values can be done using `get_iattr_value` and `has_iattr_value` of `SmartPlugin`.

File: influxdb/__init__.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+        fields.update( self.fields ) # + plugin.conf fields
+        fields.update( config['fields'] ) # + item's fields
+        fields[config['value_field']] = float( item() )
+
+        line = self.create_line(name, tags, fields)
+        self.send( line )
+        return None
+
+    def _update_values(self):
+        return None
```
Comment (ohinckel) [inline influxdb/__init__.py]: Is this method use somewhere in the code?

File: influxdb/__init__.py (hunk 3 above)
Comment (hotzen) [inline influxdb/__init__.py]: sorry, want to "cross-compile" this for callidomus as well

**Comment by hotzen** [comment]
fixed your remarks, thanks for checking this carefully!

**Comment by ohinckel** [comment]
Thanks for contributing, too.

**Pull request merged**